dist/docker/redhat/scyllasetup.py

```diff
@@ -151,5 +152,22 @@ def arguments(self):
 
         args += ["--blocked-reactor-notify-ms 999999999"]
 
+        argument = None
+        for arg in self._extra_args:
```

Comment:
Is this transformation really needed? Can't we just append `extra_arguments` to `args`?

Reply:
In base cases we can drop it in favor of simple .split .

Tested with: 
--new_arg1=val --new_arg2 val --new_arg3="val" --new_arg4 "val" --new_arg5
Result:
SCYLLA_DOCKER_ARGS="--overprovisioned --listen-address 127.0.1.1 --rpc-address 127.0.1.1 --seed-provider-parameters seeds=127.0.1.1 --blocked-reactor-notify-ms 999999999 --new_arg1=val --new_arg2 val --new_arg3=val --new_arg4 val --new_arg5"